### DISCUSSION TITLE
Can I have multiple load functions on page.server.ts?

### DISCUSSION THREAD

**Discussion by taishi55** [sveltejs/kit#7520] [Q&A]
https://kit.svelte.dev/docs/types#sveltejs-kit-serverloadevent

According to the doc above, I can see that you can include only one `invalidate` in a `load` function. 

How can I define multiple load functions for triggering multiple `invalidate ` events on +page.svelte?

**Comment by teemingc** [comment]
You could define 4 different load functions for a page (although this seems a bit hacky to fit your use case).
1. +layout.server.js
2. +page.server.js
3. +layout.js
4. +page.js
